### PR TITLE
downloaded logo and font files

### PR DESCRIPTION
**Pull request by KevinLaCarrubba**
- Added Logo .svg and .png files
- Made a white font version of the logo.
- Added Custom Font Families 
- Added Color Paletee
> 56thStreet Regular
> Gortdita Regular
- Removed about section from Landing Page
- Adjusted layout and color on Services and CTA Section
- Reworked Layout on footer
- Reworked About page layout
- Removed CTA from services page

### CODE REVIEW

**Comment by arfusop** [comment]
Task linked: [CU-863fzarya CSS Overhaul on Current Site](https://app.clickup.com/t/863fzarya)

**Comment by KevinLaCarrubba** [comment]
God help me

File: components/button/Button.styles.tsx [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+    color: var(--Off-brand-color);
+    font-weight: 800;
+    transition: 0.3s ease-in-out;
+
+    &:hover {
+        background-color: var(--Primary-accent-color);
+        border: 2px solid var(--Primary-accent-color);
+        border-radius: 5px;
+        color: var(--Primary-brand-color);
+        font-weight: 800;
```
Comment (arfusop) [inline components/button/Button.styles.tsx:18]: `cursor: pointer`

**Review by arfusop — CHANGES_REQUESTED** [review]
- The `AJ-Headshot.jpg` is definitely a catfish
- None of the buttons have the right hover cursor styling, `cursor: pointer`
### Few things on the About Page
Screenshot 2023-02-02 at 1 01 42 PM
- We want even space between all paddings and margins, so between the subheader and the actual content below needs a bit more space
- More space between profile picture and information text
- More space (evenly distributed) between the profile pic and the 2 text items below

- Weird thing on the Blog page, the nav isn't working, same for the FAQ page. The only reason i bring this up here is the nav should be styled at baseline, not based on which pages it's own. If it's showing on that given page, it should be styled. and the styles should be contained to the actual nav, not a container that only lives on each page

File: components/button/Button.styles.tsx (hunk 1 above)
Comment (KevinLaCarrubba) [reply components/button/Button.styles.tsx:18]: this button needs a rework . hate how it looks and functions . i play later tho in a Fix the Ugly Button  ticket

**Review by arfusop — APPROVED** [review]

**Pull request merged**